packages/utils/src/carequality/report.ts

```diff
@@ -0,0 +1,45 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on tops
+
+import fs from "fs";
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import {
+  generatePatientDiscoveryReport,
+  generateDocumentQueryReport,
+  generateDocumentRetrievalReport,
+} from "@metriport/core/external/carequality/ihe-gateway-v2/report/report";
+import { initReadonlyDbPool } from "@metriport/core/util/pg";
+
+async function main() {
+  const sqlDBCreds = getEnvVarOrFail("DB_CREDS");
+  const readReplicaEndpoint = getEnvVarOrFail("DB_READ_REPLICA_ENDPOINT");
+  const pg = initReadonlyDbPool(sqlDBCreds, readReplicaEndpoint);
+
+  try {
+    const [patientDiscoveryReport, documentQueryReport, documentRetrievalReport] =
+      await Promise.all([
+        generatePatientDiscoveryReport(pg),
+        generateDocumentQueryReport(pg),
+        generateDocumentRetrievalReport(pg),
+      ]);
+
+    fs.writeFileSync(
+      "./runs/carequality-report/patient-discovery-report.json",
```

Comment:
NABD, but `carequality-report` is tied to a specific HIE, but this is IHE GW, which in theory could be other HIEs in the mid-term.

Reply:
Also, no date on the file name?